pkg/operator/controller/dns_status_test.go

```diff
@@ -19,46 +19,72 @@ func TestDNSStatusConditions(t *testing.T) {
 		availDNS, desireDNS int32
 		haveNR              bool
 		availNR, desireNR   int32
+		isManaged           bool
```

Comment:
Besides `Managed` or `Unmanaged`, `ManagementState` also could be `Force`, `Removed`, or unset.